spec/registrant_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+    before(:each)do
+    @registrant_1 = Registrant.new('Bruce', 18, true)
+    @registrant_2 = Registrant.new('Penny', 15)
+    end
+    
+    describe '#initialize' do
+      it 'can initialize' do
+      expect(@registrant_1).to be_a Registrant
+      expect(@registrant_2).to be_a Registrant
+      end
+  end
+
+  it 'has the correct name' do
+    expect(@registrant_1.name).to eq('Bruce')
+    expect(@registrant_2.name).to eq('Penny')
+  end
+
+  it 'has the correct age' do
+    expect(@registrant_1.age).to eq(18)
+    expect(@registrant_2.age).to eq(15)
+  end
+
+  it 'has a permit' do
+    expect(@registrant_1.permit).to eq(true)
+    expect(@registrant_2.permit).to eq(false)
+  end
+
```

Comment:
We also want to make sure we have a test for `permit?` - it would look like your 'it has a permit' test, but would verify that the `permit?` method is working.